### PR TITLE
Catboost, warm-up FT, fine tune from pretrained FT models

### PR DESCRIPTION
**Pull request by giollimirgia**

### CODE REVIEW

File: scripts/finetune_presto.py [hunk 1]
```diff
@@ -219,6 +233,21 @@ def main(args):
         num_workers=hyperparams.num_workers,
     )
 
+    # Warmup: run a few batches with a very low learning rate to avoid destroying pretrained weights
+    logger.info("Starting warmup phase...")
+    warmup_epochs = 5
```
[the remark this answers is not in the document]
Comment (giollimirgia) [reply scripts/finetune_presto.py:238]: just because this is what i used to do in past projects. thought with such a lower lr was still ok, but ofc i can lower it
